### PR TITLE
[trivial] fix ahub issue @open sesame 07/19 16:19

### PR DESCRIPTION
**Pull request by lhs8928**
- Added try catch statement
 - Delete structually dead code

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

**Review by DonghakPark — APPROVED** [review]
LGTM

File: Applications/VGG/jni/main.cpp [hunk 1]
```diff
@@ -106,16 +106,35 @@ int main(int argc, char *argv[]) {
...
 
-  auto dataset_train = ml::train::createDataset(
-    ml::train::DatasetType::GENERATOR, trainData_cb, train_user_data.get());
-  auto dataset_valid = ml::train::createDataset(
-    ml::train::DatasetType::GENERATOR, validData_cb, valid_user_data.get());
+  std::unique_ptr<ml::train::Dataset> dataset_train;
+  try {
+    dataset_train = ml::train::createDataset(
+      ml::train::DatasetType::GENERATOR, trainData_cb, train_user_data.get());
+  } catch (...) {
```
Comment (songgot) [inline Applications/VGG/jni/main.cpp:113]: How about using exception &e
} catch (const std::exception &e) {
    std::cerr << "Error creating dataset" << e.what() << std::endl;
    return 1;
}

**Review by songgot — APPROVED** [review]

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by songgot — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**